eslint.config.mjs

```diff
@@ -1,48 +1,101 @@
 import base, { createConfig } from '@metamask/eslint-config';
-import nodejs from '@metamask/eslint-config-nodejs';
 import jest from '@metamask/eslint-config-jest';
+import nodejs from '@metamask/eslint-config-nodejs';
 import typescript from '@metamask/eslint-config-typescript';
 
-const config = createConfig(
+const NODE_LTS_VERSION = 22;
+
+const config = createConfig([
+  ...base,
   {
     ignores: [
-      'yarn.lock',
-      '**/**.map',
-      '**/**.tsbuildinfo',
-      '**/*.json',
-      '**/*.md',
-      '**/LICENSE',
-      '**/*.sh',
-      '**/.DS_Store',
       '**/dist/**',
       '**/docs/**',
       '**/coverage/**',
       'merged-packages/**',
       '.yarn/**',
       'scripts/create-package/package-template/**',
-      'apps/storybook-react-native/.storybook/FontLoader.js',
-      'apps/storybook-react-native/.storybook/config.js',
+      /**
+       * Design System specific ignores
+       * TODO review these ignores and re-enable them
+       */
```

Comment:
I've created tickets to resolve these file ignores which we should prioritize 
- https://github.com/MetaMask/metamask-design-system/issues/657
- https://github.com/MetaMask/metamask-design-system/issues/658